requirements.in

```diff
@@ -1,5 +1,5 @@
 click
-cookiecutter
+cookiecutter==1.7.0
```

Comment:
Why is this version pinned? Is the reason for this documented somewhere?

Reply:
See commit message for "Bulk update dependencies"